src/app/public/types/cell-class.ts

```diff
@@ -0,0 +1,6 @@
+export const enum SkyCellClass {
```

Comment:
Using `const` with `enum` has a few risks. If a specific value isn't used in the library's source code, that value will not be included in the bundle. I would remove the `const` for now.